client/App/pages/LandingPage/components/Header/VerticalCarousel/VerticalCarousel.module.scss

```diff
@@ -0,0 +1,53 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
how this element have 100%width if there are 2 elements in row

Reply:
If I remove `width: 100%` element of the left side will grow to maximum, so I used width to ensure that VerticalCarousel will have enough space. Should I use flex-glow or smth on wrapper of this 2 element?